File: app/src/main/java/com/comicsopentrends/model/Location.java
package com.comicsopentrends.model;

public class Location{
	private boolean isCountry;
	private String countryCode;
	private String name;
	private int id;

	public void setIsCountry(boolean isCountry){
		this.isCountry = isCountry;
	}

	public boolean isIsCountry(){
		return isCountry;
	}

	public void setCountryCode(String countryCode){
		this.countryCode = countryCode;
	}

	public String getCountryCode(){
		return countryCode;
	}

	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return name;
	}

	public void setId(int id){
		this.id = id;
	}

	public int getId(){
		return id;
	}

	@Override
 	public String toString(){
		return 
			"Location{" + 
			"isCountry = '" + isCountry + '\'' + 
			",countryCode = '" + countryCode + '\'' + 
			",name = '" + name + '\'' + 
			",id = '" + id + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/comicsopentrends/model/Location.kt
package com.comicsopentrends.model

class Location {
    var isIsCountry: Boolean = false
    var countryCode: String? = null
    var name: String? = null
    var id: Int = 0

    override fun toString(): String {
        return "Location{" +
                "isCountry = '" + isIsCountry + '\''.toString() +
                ",countryCode = '" + countryCode + '\''.toString() +
                ",name = '" + name + '\''.toString() +
                ",id = '" + id + '\''.toString() +
                "}"
    }
}
